tests/Api/Response/Parser/FiltersParserTest.php

```diff
@@ -346,6 +346,7 @@ public function parseDataProvider()
                                 'position' => 'item',
                                 'count' => '25',
                                 'image' => 'https://test.com/vendor/a_amp_c_design.jpg',
```

Comment:
Why do we have an image and an image URL? Isn't this the same?

Reply:
The difference is that imageUrl doesn't always get added, depending on what sort of value the image is. I wouldn't know all the details on how and why it's done like this though.